protected/views/patient/_form_address.php

```diff
@@ -60,8 +60,11 @@
 <div class="row field-row">
     <div class="large-3 column"><?php echo $form->labelEx($address,'country_id'); ?></div>
     <div class="large-4 column end">
-
-        <?php echo $form->dropDownList($address,'country_id', $countries, array('options'=>array(array_search('Australia', $countries)=>array('selected'=>true)))); ?>
+        <?php if (isset($address["country_id"])){
+            echo $form->dropDownList($address,'country_id', $countries, array('options'=>array($address["country_id"]=>array('selected'=>true))));
+        }else{
```

Comment:
Whitespace needs to be fixed here. Should be a space separating braces from keywords and conditions